presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java

```diff
@@ -365,6 +365,39 @@ public boolean isLimitGuaranteed(ConnectorSession session)
         return true;
     }
 
+    @Override
+    public void setColumnComment(JdbcIdentity identity, JdbcTableHandle handle, JdbcColumnHandle column, Optional<String> comment)
+    {
+        try (Connection connection = connectionFactory.openConnection(identity)) {
+            ResultSet resultSet = getColumns(handle, connection.getMetaData());
+
+            String colDefintion = "";
```

Comment:
Please avoid abbreviation. 
```suggestion
            String columnDefinition = "";
```